### DISCUSSION TITLE
Adding data or notes to a shape

### DISCUSSION THREAD

**Discussion by Scott-GT** [jgraph/drawio#4671] [Q&A]
Hi All,

Just starting to get to grips with Drawio, It's a great tool.

I was wondering if there is a way to add a note or additional data to a shape. I see I can add a URL link but I would like to add a couple of sentences of information about a shape or link between shapes i.e. Shape of a server and another of a backup tape with a link between them. I'd like to add some notes about the backup schedule or bandwidth.

Thanks

Scott

**Comment by risingfish** [comment]
If you right click on a shape there is an 'Edit Data' option that allows you to input key/value pairs.
